src/screens/Profile/ProfileScreen.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { CompositeScreenProps } from '@react-navigation/native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import {
+  Keyboard,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  Text,
+  View,
+  TextInput,
+  Alert,
+} from 'react-native';
+import { launchImageLibrary } from 'react-native-image-picker';
+import { MainStackParams, RootStackParams } from '../../Navigator';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { TableCell } from '../../components/TableCell';
+import { spacing } from '../../util/spacing';
+import { accountLogout } from '../../store/thunks/accountLogout';
+import { colors } from '../../util/colors';
+import Avatar from '../../components/Avatar/Avatar';
+import { NavIconButton } from '../../components/NavIconButton';
+import { selectMailBoxAddress } from '../../store/selectors/email';
+import { Input } from '../../components/Input';
+import { accountUpdate } from '../../store/thunks/account';
+import styles from './styles';
+
+export type ProfileScreenProps = CompositeScreenProps<
+  NativeStackScreenProps<MainStackParams, 'profile'>,
+  NativeStackScreenProps<RootStackParams>
+>;
+
+export const ProfileScreen = ({ navigation }: ProfileScreenProps) => {
+  const dispatch = useAppDispatch();
+  const [displayName, setDisplayName] = useState<string>('');
+  const [avatar, setAvatar] = useState<string>('');
+  const [isEditing, setIsEditing] = useState<boolean>(false);
+  const [isImagePreparing, setIsImagePreparing] = useState<boolean>(false);
+  const inputRefVerify = useRef<TextInput>(null);
+
+  const account = useAppSelector(state => state.account);
```

Comment:
It will re-render on each dispatch, derive data from redux and create memoized selector

Reply:
"account" field is an object and on every dispatch will re-render the screen(shallow equal). Use separate selector for each field derived from account